Tests fail: OSError: ARTS_BUILD_PATH environment variable required to locate ARTS API.

## Bug report

All tests fail due to not set arts build path.

```
arts/__init__.py:7: in <module>
    from typhon.arts import xml  # noqa
arts/xml/__init__.py:14: in <module>
    from . import read
arts/xml/read.py:15: in <module>
    from .. import types
arts/types.py:13: in <module>
    from .covariancematrix import (CovarianceMatrix)
arts/covariancematrix.py:4: in <module>
    from typhon.arts.workspace.api import arts_api
arts/workspace/api.py:44: in <module>
    + " locate ARTS API.")
E   OSError: ARTS_BUILD_PATH environment variable required to locate ARTS API.
```

--- a/typhon/arts/covariancematrix.py
+++ b/typhon/arts/covariancematrix.py
@@ -1,7 +1,6 @@
 import numpy as np
 import scipy as sp
 import matplotlib.pyplot as plt
-from typhon.arts.workspace.api import arts_api
 from typhon.arts.catalogues import Sparse
 import ctypes as c
 
@@ -172,6 +171,8 @@ class CovarianceMatrix(object):
 
 
         """
+        from typhon.arts.workspace.api import arts_api
+
         n_blocks = s.dimensions[0]
         n_inv_blocks = s.dimensions[1]
 

